api/patients.go

```diff
@@ -60,7 +60,14 @@ func (h *Handler) ListPatients(ec echo.Context, clinicId ClinicId, params ListPa
 		return err
 	}
 
-	return ec.JSON(http.StatusOK, NewPatientsResponseDto(list))
+	clinicPatientsCount, err := h.Patients.Count(ctx, &patients.Filter{
```

Comment:
Would it be easier/cheaper to go fetch the count available in the corresponding clinic object? https://github.com/tidepool-org/clinic/blob/master/clinics/clinics.go#L105

Reply:
It will be cheaper, but incorrect. The count in the clinic object excludes the demo patient account (James Jellyfish).